Faq/faq.mixer

```diff
@@ -51,7 +51,7 @@
         <a href="#2.1">2.1: Where can I download <mixer var="GAP"/></a>?
       </li>
       <li>
-        <a href="#2.2">2.2: I cannot FTP that much data. Can I get a CD?</a>
+        <a href="#2.2">2.2: I (obsolete question, removed)</a>
```

Comment:
First of all, `I` is part of the question, so it should be removed.

Second, I don't see any harm in keeping this question. If there will be a genuine case, we will be happy to look if we can help. I would only replace the word `FTP` by `download`.

Reply:
Of course we would be willing to help. But this is an *F*AQ, and I strongly doubt that this question is asked frequently